src/MongoDB.Driver.Core/ChangeStreamUpdateDescription.cs

```diff
@@ -35,12 +37,27 @@ public sealed class ChangeStreamUpdateDescription
         /// </summary>
         /// <param name="updatedFields">The updated fields.</param>
         /// <param name="removedFields">The removed fields.</param>
+        [Obsolete("Use the overload with truncatedArrays.")]
         public ChangeStreamUpdateDescription(
             BsonDocument updatedFields,
-            string[] removedFields)
+            string[] removedFields) : this(updatedFields, removedFields, null)
```

Comment:
nit: put the call to `base` or `this` on its own line:

    public ChangeStreamUpdateDescription(
        BsonDocument updatedFields,
        string[] removedFields)
        : this(updatedFields, removedFields, null)

Reply:
done